src/pages/Profile.jsx

```diff
@@ -89,24 +106,43 @@ const Profile = () => {
 
         <ProfileComponent clientProfile={clientProfile} />
 
-        <Button
-          variant="contained"
-          color="secondary"
-          size="small"
-          aria-label="Add Client Button"
-          startIcon={<AddIcon />}
-          onClick={() => setShowModal(true)}
-        >
-          Add Document
-        </Button>
-        <UploadDocumentModal showModal={showModal} setShowModal={setShowModal} />
-        <DocumentTable />
         {!client && (
           <>
-            <SetAclPermsDocContainerForm />
-            <SetAclPermissionForm />
+            <Container sx={{ display: 'flex', justifyContent: 'center', gap: 2 }}>
+              <Button
+                variant="contained"
+                color="primary"
+                size="small"
+                aria-label="Set Container Permissions Button"
+                startIcon={<SettingsIcon />}
+                onClick={() => setShowAclPermsDocContainerModal(true)}
+              >
+                Container Permissions
+              </Button>
+              <Button
```

Comment:
While I understand moving the `Container Permissions` button into the conditional rendering, why was the `Add Document` buttons moved into here as well? When the permission settings get fixed, case workers should be able to help clients upload into their Pods.

Reply:
Sorry, I moved the whole container in there without thinking.